Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UwpAppxBuildTools.cs

```diff
@@ -206,6 +215,37 @@ private static async Task<string> FindMsBuildPathAsync()
             return string.Empty;
         }
 
+        private static bool UpdateVSProj(IBuildInfo buildInfo)
```

Comment:
It's a pity that it is necessary to patch existing project files instead of handling that during project generation.
Do we also need to modify any templates for the MSBuild process?

Reply:
>It's a pity that it is necessary to patch existing project files instead of handling that during project generation.

Yeah, as far as I can tell this is a known issue. I'm not sure there's anything we can do for the time being.

> Do we also need to modify any templates for the MSBuild process?

I'm not sure, I didn't try it. I'm not sure if the MSBuild stuff works at all for Unity2019 or arm64